### PR TITLE
add function is_prime which checks to see if value is prime

### PR DESCRIPTION
**Pull request by Pratere**

### CODE REVIEW

**Comment by antoffigueroa** [comment]
I would like to review this

File: Prater/template/simple_functions.py [hunk 1]
```diff
@@ -10,3 +10,10 @@ def factorial(value):
         return 1
     else:
         return value * factorial(value - 1)
+
+
+def is_prime(value):
```
Comment (antoffigueroa) [inline Prater/template/simple_functions.py:15]: Great solution!

**Review by antoffigueroa — COMMENTED** [review]
I really liked the approach you used to solve this problem!

**Comment by astrofrog** [comment]
Thanks for opening the pull request and for the review(s)!  We will merge this shortly.  Below I have provided a few comments about the *process* of the pull request and review. But note that we are not conducting a review of the code itself, because the exercise was about the overall process not the details of the code.

* @Pratere - it would have been good to provide a description for the PR, even if it duplicates a little the title.

* @antoffigueroa - you did not "approve" or "request changes" when you submitted the review.  Without that a maintainer isn't sure if your review should be taken as a complete review or just general comments. In this case, it seems like approving would have been appropriate :)

**Pull request merged**